apps/api/src/server.ts

```diff
@@ -61,104 +60,88 @@ export const createServer = () => {
         next();
       }
     })
-    .get('/sync-prereqs', async (req, res, next) => {
+    .get('/sync-context', async (req, res, next) => {
       try {
         const userId = getUserIdFromRequest(req);
-        let isSyncOn = false;
 
-        try {
-          const { data: settings } = await axios.get(`${baseUrl}/provider/users/${userId}/settings`, {
-            headers: {
-              authorization: `Bearer ${ACCESS_TOKEN}`,
-            },
-          });
-
-          isSyncOn = settings?.isSyncOn;
-        } catch (err) {
-          // do nothing
-        }
-
-        const { data } = await axios.get(`${baseUrl}/provider/data-profiles/latest/datapoints`, {
+        const { data: context } = await axios.get(`${baseUrl}/provider/users/${userId}/sync-context`, {
           headers: {
             authorization: `Bearer ${ACCESS_TOKEN}`,
           },
         });
 
-        const { datapoints } = data;
-
-        if (!datapoints) {
-          throw new Error('dataProfile must be configured');
-        }
-
-        return res.json({
-          isSyncOn,
-          datapoints,
-        });
+        return res.json(context);
       } catch (e) {
         next(e);
       }
     })
-    .post('/user-consent', async (req, res, next) => {
+    .post('/prepare-sync', async (req, res, next) => {
       try {
+        // Pick User ID
+        //  - Pick an identifier that will serve as a unique external user id within Portabl Core™ API;
+        //  - Subject to implementation of your auth server, you might consider passing and validating you native auth params, and derive a user id from them;
         const userId = getUserIdFromRequest(req);
-        await axios.put(
-          `${baseUrl}/provider/users/${userId}`,
-          {
-            settings: {
-              isSyncOn: true,
-            },
-          },
-          {
-            headers: {
-              authorization: `Bearer ${ACCESS_TOKEN}`,
-            },
-          },
-        );
 
-        // Make a request to get claims from internal APIs
-        const claims = MOCKED_CLAIMS;
-
-        // Make a request to get the native user id from internal APIs
+        // Integrate [Update or Create a User](https://docs.getportabl.com/api-ref#update-or-create-a-user) endpoint
+        //  - You shall enable data syncronization for a user by passing the following request body:
+        //    - `{ settings: { isSyncOn: true } }`
+        //  - If user is not found by (linked to) your Portabl account yet, it will create user in (link to) your Portabl account;
```

Comment:
Can we remove the (linked to) and (link to), I think it makes sense without the additional context. Had to re-read that a few times to to understand the parens

Reply:
Agreed -- will simplify this section entirely.